### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RameshRavone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,27 +77,47 @@ public:
 
         static void _register_methods() {
            register_method("method", &SimpleClass::method);
+	   
+	   /**
+	    * How to register exports like gdscript
+	    * export var _name = "SimpleClass"
+	    **/
+	   register_property((char *)"base/name", &SimpleClass::_name, Variant("SimpleClass"));
```
Comment (karroffel) [inline README.md]: The default value must have the se type as the member pointer but here it's Variant. If it is a String then you must put a String there

Comment (RameshRavone) [inline README.md]: Updated, was really in hurry, you should wait until godot_header README is updated as creating `.gdns` is left out.

File: README.md [hunk 2]
```diff
@@ -26,40 +26,35 @@ $ cd cpp_bindings
 
 Edit `SConstruct` file and assign your godot executable path at line:7 `godot_bin_path = "../godot_fork/bin/"`,
 
-Building core/bindings, (`libgodot_cpp_core.so` , `libgodot_cpp_bindings.so`)
+Building bindings in linux,(`libgodot_cpp_bindings.a`)
 ```
 $ scons p=linux
```
Comment (karroffel) [inline README.md]: This line isn't needed anymore as everything gets compiled at once.

Comment (RameshRavone) [inline README.md]: sure,

**Pull request merged**